container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/implementations/vector/vector_configuration_creator_factory.go

```diff
@@ -8,7 +8,8 @@ func createVectorConfigurationCreatorForKurtosis(
 	listeningPortNumber uint16,
 	httpPortNumber uint16,
 	sinks logs_aggregator.Sinks,
+	shouldTurnOffPersistentVolumeLogsCollection bool,
```

Comment:
Btw if you do choose to go for positive vs negative, be careful doing the refactor. If you just start renaming the variable and don't trace down where it gets used, the IDE won't throw an error (b/c it's still a bool variable) but you'll accidentally be using the inverted version